ipfx/feature_vectors.py

```diff
@@ -306,7 +306,7 @@ def isi_shape(sweep_set, features, n_points=100, min_spike=5):
         end_index = tsu.find_time_index(isi_sweep.t, isi_sweep.t[fast_trough_index] + 0.1)
         isi_raw = isi_sweep.v[fast_trough_index:end_index] - threshold_v
 
-        width = len(isi_raw) / n_points
+        width = len(isi_raw) // n_points
```

Comment:
Just as a double check, this *should* be truncating division?

Reply:
Yes, this should be an integer division, because the result should be an integer.